website/docs/docs/build/metrics-overview.md

```diff
@@ -76,19 +82,48 @@ metrics:
       {{  Dimension('entity__name') }} > 0 and {{ Dimension(' entity__another_name') }} is not
       null and {{ Metric('metric_name', group_by=['entity_name']) }} > 5
 ```
+</File>
 
 </VersionBlock>
 
-This page explains the different supported metric types you can add to your dbt project.
-
 import SLCourses from '/snippets/_sl-course.md';
 
 <SLCourses/>
 
-### Conversion metrics
+## Default granularity for metircs
+
+It's possible to define a default time granularity for metrics that differs from the granularity of the default aggregation time dimensions (`metric_time`). This is useful if your time dimension has a very fine grain, like second or hour, but you typically query metrics rolled up at a coarser grain. The granularity can be set using the `time_granularity` parameter on the metric and defaults to `day`.
```

Comment:
Would note that while it defaults to day, if day is not available because the dimension is defined at a coarser granularity, it will default to the defined granularity for the dimension!